11-payment-encoding.md

```diff
@@ -175,13 +182,22 @@ A writer:
         `fee_base_msat`, `fee_proportional_millionths`, and `cltv_expiry_delta` are as
         specified in [BOLT #7](07-routing-gossip.md#the-channel_update-message).
     - MAY include more than one `r` field to provide multiple routing options.
+  - if `9` contains non-zero bits:
```

Comment:
Should be on the same format as the other fields? `MAY include one `9` field.`

Reply:
No, it either MUST or MUST NOT.